README.md

```diff
@@ -99,31 +109,28 @@ trigger formatters via key mappings and/or autocommand hooks. See
 vroom/main.vroom to learn more about formatting features, and see
 vroom/FORMATTER-NAME.vroom to learn more about usage for individual formatters.
 
-Most formatters have some options available that can be configured via Glaive.
-You can get a quick view of all codefmt flags by executing `:Glaive codefmt`, or
-start typing flag names and use tab completion. See `:help Glaive` for usage
-details.
-
 ## Creating a New Formatter
 
 Assume a filetype `myft` and a formatter called `MyFormatter`. Our detailed
 guide to creating a formatter [lives
 here](https://github.com/google/vim-codefmt/wiki/Formatter-Integration-Guide).
 
-*   Create an issue for your new formatter and discuss! Once there's consensus,
-    continue onward.
+*   Create an issue for your new formatter and discuss!
 
 *   Create a new file in `autoload/codefmt/myformatter.vim` See
     `autoload/codefmt/buildifier.vim for an example. This is where all the
     logic for formatting goes.
 
-*   Register the formatter with:
+*   Register the formatter in
+    [plugin/register.vim](https://github.com/google/vim-codefmt/blob/master/plugin/register.vim) 
+    with:
 
     ```vim
     call s:registry.AddExtension(codefmt#myformatter#GetFormatter())
     ```
 
-*   Create a flag in instant/flags.vim
+*   Create a flag in
+    [instant/flags.vim](https://github.com/Kashomon/vim-codefmt/blob/master/instant/flags.vim)
```

Comment:
Yeah, it does. Just verified using the Preview feature in GitHub's web editor. It's worth using that style because (a) it will link to the appropriate branch/fork/commit wherever you use it, and (b) it fits on one line that way.